<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="natalia-All-Series" name="myobject.Incident" tests="1" failures="0" timestamp="15 Sep 2015 07:46:06 GMT" time="33.780" errors="1">
  <testcase name="testDelete" time="33.780" classname="myobject.Incident">
    <error type="java.lang.AssertionError" message="Ошибка при проверке значения поля Место инцидента (производство, участок, цех)
org.openqa.selenium.NoSuchElementException: no such element
  (Session info: chrome=39.0.2171.95)
  (Driver info: chromedriver=2.14.313457 (3d645c400edf2e2c500566c9aa096063e707c9cf),platform=Linux 3.13.0-43-lowlatency x86) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 30.02 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.45.0&#039;, revision: &#039;5017cb8e7ca8e37638dc3091b2440b90a1d8686f&#039;, time: &#039;2015-02-27 09:10:26&#039;
System info: host: &#039;natalia-All-Series&#039;, ip: &#039;127.0.0.1&#039;, os.name: &#039;Linux&#039;, os.arch: &#039;i386&#039;, os.version: &#039;3.13.0-43-lowlatency&#039;, java.version: &#039;1.7.0_65&#039;
*** myobject.Element info: {Using=xpath, value=//div[@id=&quot;workspace&quot; or @class=&quot;workspace&quot;]//node()[contains(text(),&quot;Данные об инциденте&quot;)]/following-sibling::node()//descendant-or-self::node()[contains(text(),&quot;Место инцидента (производство, участок, цех)&quot;)]/../following-sibling::div}
Session ID: cb35cffd0f927622b50ac4eed8be3538
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=LINUX, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=/tmp/.com.google.Chrome.SifOe6}, rotatable=false, locationContextEnabled=true, mobileEmulationEnabled=false, version=39.0.2171.95, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false, webStorageEnabled=true, nativeEvents=true, applicationCacheEnabled=false, takesScreenshot=true}]">
      <![CDATA[java.lang.AssertionError: Ошибка при проверке значения поля Место инцидента (производство, участок, цех)
org.openqa.selenium.NoSuchElementException: no such element
  (Session info: chrome=39.0.2171.95)
  (Driver info: chromedriver=2.14.313457 (3d645c400edf2e2c500566c9aa096063e707c9cf),platform=Linux 3.13.0-43-lowlatency x86) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 30.02 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.45.0', revision: '5017cb8e7ca8e37638dc3091b2440b90a1d8686f', time: '2015-02-27 09:10:26'
System info: host: 'natalia-All-Series', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'i386', os.version: '3.13.0-43-lowlatency', java.version: '1.7.0_65'
*** myobject.Element info: {Using=xpath, value=//div[@id="workspace" or @class="workspace"]//node()[contains(text(),"Данные об инциденте")]/following-sibling::node()//descendant-or-self::node()[contains(text(),"Место инцидента (производство, участок, цех)")]/../following-sibling::div}
Session ID: cb35cffd0f927622b50ac4eed8be3538
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{platform=LINUX, acceptSslCerts=true, javascriptEnabled=true, browserName=chrome, chrome={userDataDir=/tmp/.com.google.Chrome.SifOe6}, rotatable=false, locationContextEnabled=true, mobileEmulationEnabled=false, version=39.0.2171.95, takesHeapSnapshot=true, cssSelectorsEnabled=true, databaseEnabled=false, handlesAlerts=true, browserConnectionEnabled=false, webStorageEnabled=true, nativeEvents=true, applicationCacheEnabled=false, takesScreenshot=true}]
	at org.testng.Assert.fail(Assert.java:94)
	at Base.MOWebdriver.revisionValue(MOWebdriver.java:636)
	at myobject.Incident.testDelete(myobject.Incident.java:329)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
]]>
    </error>
  </testcase> <!-- testDelete -->
</testsuite> <!-- myobject.Incident -->
